stress-tests/base/cronjob.yaml

```
@@ -0,0 +1,87 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: stress-tests
+  labels:
+    app: thoth
+    component: stress-tests
+spec:
+  schedule: "@weekly"
+  suspend: true
+  successfulJobsHistoryLimit: 5
+  failedJobsHistoryLimit: 4
+  concurrencyPolicy: Forbid
+  startingDeadlineSeconds: null
+  jobTemplate:
+    spec:
+      backoffLimit: 0
+      template:
+        metadata:
+          labels:
+            app: thoth
+            component: stress-tests
+        spec:
+          restartPolicy: Never
+          containers:
+            - image: "stress-tests:latest"
+              name: stress-tests
+              env:
+                - name: MAIL_REPORT
+                  valueFrom:
+                    configMapKeyRef:
+                      key: mail-report
+                      name: stress-tests
+                - name: GENERATE_REPORT
+                  valueFrom:
+                    configMapKeyRef:
+                      key: generate-report
+                      name: stress-tests
+                - name: THOTH_DEPLOYMENT_NAME
+                  valueFrom:
+                    configMapKeyRef:
+                      key: deployment-name
+                      name: stress-tests
+                - name: THOTH_LOGGING_NO_JSON
+                  valueFrom:
+                    configMapKeyRef:
+                      name: thoth
+                      key: logging-no-json
+                - name: THOTH_USER_API_HOST
+                  valueFrom:
+                    configMapKeyRef:
+                      key: user-api
+                      name: stress-tests
+                - name: THOTH_MANAGEMENT_API_HOST
+                  valueFrom:
+                    configMapKeyRef:
+                      key: management-api
+                      name: stress-tests
+                - name: THOTH_AMUN_API_HOST
+                  valueFrom:
+                    configMapKeyRef:
+                      key: amun-api
+                      name: stress-tests
+                - name: THOTH_MANAGEMENT_API_SECRET
+                  valueFrom:
+                    secretKeyRef:
+                      key: management-api-token
+                      name: stress-tests
```

```suggestion
                      name: thoth
```

Lets take it from thoth secret, which is available in the cluster namespace
later we can switch it if required.